src/config/NetworkConfig.ts

```diff
@@ -93,10 +93,10 @@ const defaultMainnetNetworkConfig: NetworkConfig = {
         namespaceGracePeriodDuration: 2592000,
         lockedFundsPerAggregate: '10000000',
         maxCosignatoriesPerAccount: 25,
-        blockGenerationTargetTime: 15,
+        blockGenerationTargetTime: 30,
         maxNamespaceDepth: 3,
         maxMosaicDuration: 21024000,
-        minNamespaceDuration: 172800,
+        minNamespaceDuration: 86400,
         maxNamespaceDuration: 2102400,
```

Comment:
"minNamespaceDuration": "30d",
"maxNamespaceDuration": "1825d",

Max value doesn't seem right to me

Also, NamespaceDuration is network properties, would it be better to calculate from network/properties?

Reply:
I believe these are the default values and is getting overwritten by the connected node's network properties.